src/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_AnyAttribute.cs

```diff
@@ -0,0 +1,202 @@
+﻿using Xunit;
+using Xunit.Abstractions;
+using System.IO;
+using System.Xml.Schema;
+
+namespace System.Xml.Tests
+{
+    public class TC_SchemaSet_AnyAttribute : TC_SchemaSetBase
+    {
+        private ITestOutputHelper _output;
+        public bool bWarningCallback;
+        public bool bErrorCallback;
+        public int errorCount;
+        public int warningCount;
+        public bool WarningInnerExceptionSet = false;
+        public bool ErrorInnerExceptionSet = false;
```

Comment:
The fields should be named consistently, e.g. without the "b" prefix and capitalized consistently. Any reason they can't be private? If they can be private, they should be prefixed with an underscore and camel case. If they need to be public, they should be pascal case.